src/main/java/org/taktik/icure/entities/Invoice.java

```diff
@@ -598,4 +600,13 @@ public String getEncryptedSelf() {
 	public void setEncryptedSelf(String encryptedSelf) {
 		this.encryptedSelf = encryptedSelf;
 	}
+
+
+	public IdentityDocumentReader getIdDocument() {
+		return idDocument;
+	}
+
+	public void setIdDocument(IdentityDocumentReader idDocument) {
+		this.idDocument = idDocument;
+	}
```

Comment:
@gpiroux @phil1618 Currently, we set the `eidReadingHour`, `eidReadingValue` in `InvoicingCodeDto`, we could set the data in the `InvoiceDto`. If you think it's better, please create a new story.

Reply:
@fthuin @gpiroux Je pense que cela vaut la peine: https://www.pivotaltracker.com/story/show/162421445